ch_pipeline/core/containers.py

```diff
@@ -802,6 +802,39 @@ def source(self):
         return self.index_map["source"]
 
 
+class GaltAutocorrelation(FreqContainer, TODContainer):
+    _axes = ("freq", "pol", "time")
```

Comment:
```suggestion
    _axes = ("pol",)
```
If you inherit from the base classes, then you no longer need to specify the axes covered by those classes.   A new instance will have all the unique axes contained in all subclasses.